site/ja/tutorials/load_data/images.ipynb

```diff
@@ -0,0 +1,3254 @@
+{
```

Comment:
<p>memo: this cell is changed in original and r2 notebook.</p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/docs/pull/427/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>Ditto.</p>